nbextensions/appCell2/main.js

```diff
@@ -55,27 +50,15 @@ define([
         );
     }
 
-    function setupWorkspace(workspaceUrl) {
```

Comment:
Like with the bulk import cell, the only thing that gets used from the workspace object is the workspace ID, which can be taken from the runtime instead. I have removed all the `workspaceInfo` references from the files that don't use the workspace. This file is now similar to `main.js` for the bulk import cell.